### ISSUE TITLE
TOML parse error on yazi.toml

### ISSUE THREAD

**Issue by jrcasey** [yazi-rs/schemas#61]
On executing `yazi` from a ghostty terminal running fish, throws two TOML parse errors (below). I've also attached my yazi.toml as a text file.

Configuration:
- Yazi 26.5.6
- MBP M4 Pro, MacOS 26.3.1
- Ghostty
- FISH shell
- Installed yazi through homebrew (brew is updated)

Behavior:
open shell -> execute `yazi` -> errors -> dismiss errors -> yazi opens with issues

Issues:
- No file preview
- Not able to open a file in editor with `o` or `<Enter>`
 
Error message:
```
TOML parse error at line 3, column 1
  |
3 | "$schema" = "https://yazi-rs.github.io/schemas/yazi.json"
  | ^^^^^^^^^
must be a kebab-cased string

Press <Enter> to continue with preset settings...

TOML parse error at line 94, column 2
   |
94 |    { name = "*/", fg = "#00d9fa" },
   |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
at least one of `url` or `mime` must be specified
```

[yazi.txt](https://github.com/user-attachments/files/27504156/yazi.txt)

**Comment by sxyazi** [comment]
Duplicate of https://github.com/sxyazi/yazi/issues/3938

As for the second problem, the error is pretty self‑explanatory: [there's no `name` property](https://yazi-rs.github.io/docs/configuration/yazi#plugin.previewers), you need to use either `url` or `mime`. 

Some context to add: v26.5.6 has made the config validation a bit stricter, so config errors like this get caught, before v26.5.6 a bad rule would just be silently ignored and treated as a no‑op.

**Comment by jrcasey** [comment]
Thanks - I ended up reinstalling after some whack-a-mole. Thanks for the quick reply!
